01-introduction-to-react/jsx-and-babel/app.jsx

```diff
@@ -0,0 +1,104 @@
+//part 1
+class FirstComponent extends React.Component {
+	render() {
```

Comment:
FYI, when it comes to company project time, we'll probably have you update your settings in your text editor to make JavaScript use 2 spaces for indentation. When you do code reviews on github, tabs look huuuuuuuge